### ISSUE TITLE
[BUG] Not possible to provide empty strings as base_parameters in jobs

### ISSUE THREAD

**Issue by evisser** [databrickslabs/dbx#170]
## Expected Behavior
When adding `param_name: ''` in a jobs deployment yaml/json file as base_parameter I would expect an empty value for that parameter for the job (i.e. when inspecting in the UI).

## Current Behavior
When adding  `param_name: ''` in a jobs deployment yaml/json file as base_parameter the value for that parameter instead becomes something like `dbfs:/dbx/project_name/96181aaa677247329df70e7a50d346f6/artifacts/.`.

## Steps to Reproduce (for bugs)
The behaviour can be reproduced using the following yaml file and then executing `dbx deploy --deployment-file ./deployment.yaml --no-rebuild -e MOD` 

deployment.yaml:
```---
environments:
  MOD:
    jobs:
      - name: "empty_param_test"
        tasks:
          - task_key: empty_param_task
            new_cluster:
              spark_version: "9.1.x-scala2.12"
              node_type_id: "Standard_F8s"
              autoscale:
                min_workers: 1
                max_workers: 8
            notebook_task:
              notebook_path: "/path/to/notebook"
              base_parameters:
               empty_param: ''
               non_empty_param: 'value'
               another_empty_param: ''
```

## Your Environment

* dbx version used: 0.3.3
* Databricks Runtime version: 9.1

**Comment by renardeinside** [comment]
Hi @evisser , 

this issue is already fixed and will be released in 0.4.0 
For the moment please take a look at [strict path adjustment](https://dbx.readthedocs.io/en/latest/path_adjustment.html)

**Comment by evisser** [comment]
Hi @renardeinside,

thanks for the quick reply. Adding the parameter mentioned in the link makes it work as expected. Good to hear that a fix is coming in 0.4.0.

**Comment by renardeinside** [comment]
closing this one since it's fixed but not yet released :)
